reference-implementation/lib/transform-stream.js

```diff
@@ -295,9 +295,18 @@ class TransformStreamDefaultSink {
   }
 }
 
-function TransformStreamDefaultSinkDefaultTransform(chunk, controller) {
-  TransformStreamDefaultControllerEnqueue(controller, chunk);
-  return Promise.resolve();
+function TransformStreamDefaultSinkInvokeTransform(stream, chunk) {
+  const controller = stream._transformStreamController;
+  const transformer = stream._transformer;
+
+  const method = transformer.transform; // throws
```

Comment:
Although I realize they are emulating the spec annotations, these comments would be clearer as "can throw", IMO.